### DISCUSSION TITLE
Switch Pattern Help text

### DISCUSSION THREAD

**Discussion by yongjie1** [twilio-labs/paste#2937] [Q&A] [answered]
For the Switch Pattern, can the help text be changed depends on the status?

See examples:
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/91665462/211939373-172178c5-47d9-4d76-a5c7-add2f3ba7620.png">

**Comment by nkrantz** [comment, marked as answer]
Hi @yongjie1! The Label is the only text that needs to remain constant in order for the Switch to be accessible. There's no problem with changing the Help Text based on state.
